public/doc/openapi-3-v1.0.0.json

```diff
@@ -717,6 +717,95 @@
                 }
             }
         },
+        "/workflows/resolve": {
+            "post": {
+                "tags": [
+                    "Workflow"
+                ],
+                "summary": "Get a list of workflows ids",
```

Comment:
summary: Get all workflows linked to a resource object.
Same for the description.